boards/feather-m0/include/board.h

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (C) 2017 Inria
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup    boards_adafruit-feather-m0 Adafruit Feather M0
+ * @ingroup     boards
+ * @brief       Support for the Adafruit Feather M0.
+ *
+ * ### General information
+ *
+ * Feather M0 boards are development boards shipped by
```

Comment:
nice detailed infos on that board, I'd rather put that into a separate `doc.txt` in the root directory of the board.

Reply:
That's a good idea.
Maybe one day, it could be used to auto-generate (by replacing some keyword) the list for provided features (i2c, gpio, etc) for this board, when generating the doxygen documentation.